ui/src/views/AutogenView.vue

```diff
@@ -211,6 +211,11 @@
                 :pagination="true"
                 style="overflow-y: auto"
               >
+                <template #bodyCell="{ column, text }">
```

Comment:
question: where do we need this?

Reply:
this was done because asnumbers and publicIP allocated time is in the incorrect format. The change was made to have them displayed in human readable format.

Reply:
thanks @Pearl1594 
cc @shwstppr